app/models/hyrax/file_metadata.rb

```diff
@@ -140,8 +139,20 @@ def valid?
       file.valid?(size: size.first, digests: { sha256: checksum&.first&.sha256 })
     end
 
+    ##
+    # @deprecated get content from #file instead
+    #
+    # @return [#to_s]
+    def content
+      Deprecation.warn('This convienince method has been deprecated. ' \
```

Comment:
deprecating this. users should get comfortable using valkyrie's interfaces.

considering deprecating `#file` for the same reason.